examples/cron/pom.xml

```diff
@@ -31,6 +31,7 @@
     <properties>
         <noDeps>true</noDeps>
         <quarkus.camel.main.routes-discovery.enabled>false</quarkus.camel.main.routes-discovery.enabled>
+        <quarkus.package.type>uber-jar</quarkus.package.type>
```

Comment:
Is this change required ?

Reply:
Not necessarily. It is related to the PR I made previously and it effectively reverts the change I introduced, generating `uber-jar` instead of the `fast-jar`:

    java -jar target/quarkus-app/quarkus-run.jar
    java -jar target/camel-k-runtime-example-cron-runner.jar

In case this is undesired, I will remove it from the PR (Apparently I should have separated it in 2 commits in the first place.)

Reply:
I'd prefer to use the fast-jar mode as it is the mode we use on camel-k (unless there are known issues)

Reply:
Understood, will remove this particular type of changes

Reply:
Should be back to the previous fast-jar usage